enterprise/backend/test/metabase_enterprise/upload_test.clj

```diff
@@ -10,51 +11,53 @@
 
 (use-fixtures :once (fixtures/initialize :db :test-users))
 
-(deftest create-disabled-for-sandboxed-user-test
+(deftest uploads-disabled-for-sandboxed-user-test
```

Comment:
I found these sandboxed user tests to be too loose, so I tightened them. The user should not be able to upload to a table if they have sandboxed access to _any_ database. Previously, `update-disabled-for-sandboxed-user-test` and `create-disabled-for-sandboxed-user-test` only tested that the user should not be able to upload to a table if they have sandboxed access for the _uploads_ database.